tests/tasks/test_tasks.py

```diff
@@ -1,21 +1,59 @@
 import logging
-
+import unittest
+from unittest import mock
+from unittest.mock import patch
+import tempfile
 from jiant.tasks.registry import REGISTRY
+from jiant.preprocess import get_tasks
+from jiant.tasks import REGISTRY
+
+
+class Record:
+    def __init__(self, task_src_path, max_seq_len, name, tokenizer_name):
+        self.example_counts = {"train": 100, "val": 90, "val": 88}
+
+    def load_data(self):
+        return
+
+    def count_examples(self):
+        return 0
+
+
+class TestTasks(unittest.TestCase):
+    def setUp(self):
```

Comment:
The fields initialized in `setUp` aren't necessarily relevant to all tests in `TestTasks`. Can you move the test-specific initializations into the relevant test case?